src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,94 @@
+
+# New Items & Abilities
+
+New content that revolves around both a higher quality of life and a more immersive playfield.
+
+## Flavour Additions
+
+- The Space Ninja now comes with an adequately themed satchel, as well as a katana sheath.
+- The Space Ninja's pinpointer will now act as software for the brand new space ninja PDA instead of being its own gadget, the agent ID card will also be within the aforementioned device.
+- Equipping the space ninja helmet is now an instantaneous toggle action that comes with the space ninja suit.
+
+## Ability Additions
+
+- The Space Ninja can now fabricate ninja caltrops, a variant of ninja throwing stars with an emphasis on piercing damage and bleeding, once deployed, that is, dropped on the ground, they will progressively get more and more camouflaged to the point of near-invisibility. Upon being stepped on, the caltrops will stick to the victim and perform continous bloodloss damage until taken off.
+- The ninja visor can now alternate between normal vision and night vision.
+
+# New Objectives & Mechanics
+
+Gives the Space Ninja more to do and tinker with, so that all rounds don't play out the same.
+
+## Objective Additions
+
+- Steal X spesos - A strongbox for Cargo containing all their deposited spesos will be located in the vault, the Space Ninja is tasked with stealing a designated amount - An alternative to "Steal X technologies".
+- Upload the law board - An AI law board with a Spider Clan-oriented law set, the Space Ninja is tasked with uploading it to the AI upload console - An alternative to "Detonate your spider charge".
+- Hack Si-XXXX - The Space Ninja's kill-target equivalent, they are tasked with hacking a specific cyborg, turning it into a zealous Spider Clan tech-priest that obsesses over hoarding all manners of electronics and machines.
+
+## Objective Changes
+
+- All stealing objectives can now only be performed with a specialized item that is dropped by the ninja upon death, unlike previously where they only used their gloves - This allows the crew to regain their losses.
+- Make an announcement - The communications computer objective now revolves around the Space Ninja sending a Spider Clan Announcement from it, the contents of said announcement being up to their discretion. Like before, it requires the use of turned on space ninja gloves, the one key difference being that now the EMAG interaction is instantaneous, overriding all access restrictions as well as unlocking Spider Clan as an announcement sender. (Replaces "Call in a threat", the new objective keeping the threat-calling functionality is still in discussion)
+- Hack X - 1-3 of the Miscellaneous Interactions machines is picked to be hacked by the Space Ninja. Note that you can still hack machines that aren't included in your objectives. (Replaces "Set Everyone to Wanted")
```

Comment:
I like this idea.

While Ninjas aren't under the meta shield on WizDen, players still behave an in odd IC way and start to defend consoles and ninja objectives, even though it's much more interesting to pretend otherwise.

I think giving ninja enough objectives will be good, as security *possibly* can't guard them all. Security will have to take a more active approach to ninja countering. They can't simply spread out and guard everything, and if they try to, the ninja gets something unguarded in return.